README.md

```diff
@@ -8,33 +8,20 @@ See [CONTRIBUTING.md](.github/CONTRIBUTING.md)
 
 ## Developing
 
-This site is built with [Jekyll](https://jekyllrb.com/docs/home/).
+This site is built with [Eleventy](https://www.11ty.dev/).
 
 ### Quick start
 
-This project relies on [Ruby](https://www.ruby-lang.org/en/), [Node](https://nodejs.org/), and [npm](https://www.npmjs.com/). Before following these steps you'll need to [set up a Ruby dev environment](https://jekyllrb.com/docs/installation/) as well as [install node and npm](https://blog.npmjs.org/post/85484771375/how-to-install-npm) if you haven't already.
+This project relies on [Node](https://nodejs.org/), and [npm](https://www.npmjs.com/). Before following these steps you'll need to [install node and npm](https://blog.npmjs.org/post/85484771375/how-to-install-npm) if you haven't already.
 
 ```sh
 git clone git@github.com:cloudfour/pwastats.git
 cd pwastats
-gem install bundler
-bundle install
 npm install
+npx @11ty/eleventy --serve
 ```
 
-For local development:
-
-```
-npm start
-```
-
-For a single build:
-
-```
-npm run build
-```
-
-View the local site at http://localhost:4000.
```

Comment:
It's important that we still document the _build_ command versus the _development start_ command. Let's revert to using package scripts to alias those as we were before.